GenderPayGap.Database/Models/AuditLog.cs

```diff
@@ -1,17 +1,31 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations.Schema;
 using GenderPayGap.Core;
 using Newtonsoft.Json;
 
 namespace GenderPayGap.Database.Models
 {
+    [JsonObject(MemberSerialization.OptIn)]
     public class AuditLog
     {
 
+        [JsonProperty]
         public long AuditLogId { get; set; }
+        [JsonProperty]
         public AuditedAction Action { get; set; }
+        [JsonProperty]
         public DateTime CreatedDate { get; set; }
+        [JsonProperty]
+        public string DetailsString { get; set; }
+
+        [JsonProperty]
+        public long? OrganisationId { get; set; }
+        [JsonProperty]
+        public long? OriginalUserId { get; set; }
+        [JsonProperty]
+        public long? ImpersonatedUserId { get; set; }
+
         public virtual Organisation Organisation { get; set; }

```

Comment:
For consistency would it be ok to add [ForeignKey("OrganisationId")]

Reply:
Ah yes, good idea